### PR TITLE
Remove reactions to post message

### PR DESCRIPTION
**Pull request by nishigori**
This proposal is slackbot can remove reaction itself.

For example, the following cases are assumed:

1. add reaction 👀 (`:eyes:`)
1. exec something few times
1. remove reaction `:eyes:`
1. and add reaction :o: (`:o:`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,9 +121,13 @@ import re
 @respond_to('hi', re.IGNORECASE)
 def hi(message):
     message.reply('I can understand hi or HI!')
-    # react with thumb up emoji
+    # add reaction with thumb up emoji
```
Comment (townie) [inline README.md]: I might make a new example... `wave`
```
@respond_to('wave')
def wave(message):
    # bot adds reaction of :wave: emoji to message
    message.react('wave')

   from time import sleep
   sleep(1)
 
    # bot removes it's reaction of :wave: emoji on message
   message.unreact('wave')
```

**Review by townie — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (nishigori) [reply README.md]: @townie I'm just adding to above example, thx 👍

**Comment by nishigori** [comment]
Oh... travis-ci job canceled.. Please retry :-p

**Comment by townie** [comment]
travis-ci job retry

**Comment by nishigori** [comment]
Sorry, I'm just force pushed for rebase latest HEAD.

**Comment by catando** [comment]
Why was this reverted? I can find no explanation...

**Pull request merged**